doc/api/wasi.md

```diff
@@ -22,15 +22,51 @@ const wasi = new WASI({
 const importObject = { wasi_snapshot_preview1: wasi.wasiImport };
 
 (async () => {
-  const wasm = await WebAssembly.compile(fs.readFileSync('./binary.wasm'));
+  const wasm = await WebAssembly.compile(fs.readFileSync('./demo.wasm'));
   const instance = await WebAssembly.instantiate(wasm, importObject);
 
   wasi.start(instance);
 })();
 ```
 
+To run the above example, create a new WebAssembly text format file i.e. `demo.wat`
```

Comment:
It appears that _i.e._ is being used to mean "for example" here but that is not what it means. I'll update that too with a fixup unless someone beats me to it. I see now that WebAssembly text is not currently supported by highlight.js and I'm trying to figure out what to do. Probably just change this to text and not worry about it.